package.json

```diff
@@ -86,7 +81,12 @@
     "@aws-cdk/aws-appsync-alpha": "*"
   },
   "dependencies": {
+    "@functionless/ast-reflection": "0.0.2",
```

Comment:
Should we start to use the second position for breaking changes and let this be `~0.0.2`?

Reply:
Does `~` semantics apply to `0.x.y`?